cogs/yourlsClient/yourlsCmd.py

```diff
@@ -2,452 +2,11 @@
-        except RuntimeError as error:
-            await ctx.send(error)
-        except HTTPError as error:
-            if error.response.status_code == 404:
-                await ctx.send(
-                    "Could not find the keyword, please make sure it is correct and try again!"
-                )
-            else:
-                self.logger.error(error, exc_info=True)
-                await ctx.send(DEFAULT_ERROR)
-        except RequestException as error:
-            self.logger.error(error)
-            await ctx.send(DEFAULT_ERROR)
-        else:
-            await ctx.send(embed=embed)
-
-    @yourlsBase.group(name="settings")
-    async def settingsBase(self, ctx: Context):
-        """Configure the YOURLS connection"""
-
-    @settingsBase.command(name="api")
-    async def api(self, ctx: Context, apiEndpoint: str):
-        """Configure the API endpoint for YOURLS.
-
-        Parameters
-        ----------
-        apiEndpoint: str
-            The URL to the YOURLS API endpoint.
-        """
-        await self.config.guild(ctx.guild).get_attr(KEY_API).set(apiEndpoint)
-        self.logger.info(
-            "%s (%s) modified the YOURLS API endpoint for %s (%s)",
-            ctx.author.name,
-            ctx.author.id,
-            ctx.guild.name,
-            ctx.guild.id,
-        )
-        await ctx.send(f"API endpoint set to {apiEndpoint}")
-
-    @settingsBase.command(name="signature", aliases=["sig"])
-    async def sig(self, ctx: Context, signature: str):
-        """Configure the API signature for YOURLS.
-
-        Parameters
-        ----------
-        signature: str
-            The signature to access the YOURLS API endpoint.
-        """
-        await self.config.guild(ctx.guild).get_attr(KEY_SIGNATURE).set(signature)
-        self.logger.info(
-            "%s (%s) modified the YOURLS signature for %s (%s)",
-            ctx.author.name,
-            ctx.author.id,
-            ctx.guild.name,
-            ctx.guild.id,
-        )
-        await ctx.send(f"API signature set.")
-        await ctx.message.delete()
-
-    async def fetchYourlsClient(self, guild: discord.Guild):
-        """Create the YOURLS client.
-
-        Parameters
-        ----------
-        guild: discord.Guild
-            The guild to look up the YOURLS API configuration for.
-
-        Returns
-        -------
-        yourls.YOURLSClient
-            The YOURLS client, which you can use to interact with your YOURLS instance.
-
-        Raises
-        ------
-        YOURLSNotConfigured
-            Unable to create the YOURLS client because of missing information.
-        """
-        api = await self.config.guild(guild).get_attr(KEY_API)()
-        sig = await self.config.guild(guild).get_attr(KEY_SIGNATURE)()
-        if not (api and sig):
-            raise YOURLSNotConfigured("Please configure the YOURLS API first.")
-        return YOURLSClient(api, signature=sig)
+class YOURLS(commands.Cog, CommandHandlers):
+    pass
```

Comment:
Can we add a docstring here instead of `pass`ing, maybe something along the lines of `Manage Your Own URL Shortener (YOURLS) from Discord`? It is used in the help string of the cog name if we run the `[p]help YOURLS` command.

Reply:
Yep. I thought about this too, but decided to post to see your thoughts.